Gruntfile.js

```diff
@@ -64,14 +65,21 @@ module.exports = function (grunt) {
         parsers: {
           '.js': 'javascript'
         }
-      });
+      })
 
       jsWalker.on('end', function () {
-        done();
-      });
-    });
-  });
+        done()
+      })
+    })
+  })
+
+  // load local Grunt tasks
+  grunt.loadTasks('tasks')
+
+  grunt.registerTask('lint',
+  'Alias for eslint tasks',
+  ['eslint'])
```

Comment:
yeah, what @vladikoff said before. Just put this all on one line. It's unlikely we'll add JSCS to this repo, and it reads easier on a single line for now.